source/beamline/AMPVControl.h

```diff
@@ -132,12 +132,16 @@ class AMReadOnlyPVControl : public AMControl {
 
 protected slots:
 	/// Handles the low limit PV changing. Converts the limit before forwarding the signal on.
-	virtual void onLowLimitPVValueChanged(double newLowLimit);
+	virtual void setLowLimitValue(double newLowLimit);
```

Comment:
Aside from the `onWritePVInitialized()` calls, the method is typically called via the signal/slot mechanism from what I can tell.